eng/build.ps1

```diff
@@ -234,7 +236,9 @@ foreach ($argument in $PSBoundParameters.Keys)
     "properties"             { $arguments += " " + $properties }
     "verbosity"              { $arguments += " -$argument " + $($PSBoundParameters[$argument]) }
     "cmakeargs"              { $arguments += " /p:CMakeArgs=`"$($PSBoundParameters[$argument])`"" }
-    "ninja"                  { $arguments += " /p:Ninja=$($PSBoundParameters[$argument])" }
+    # The -ninja switch is a no-op since Ninja is the default generator on Windows.
+    "ninja"                  { }
+    "msbuild"                { $arguments += " /p:Ninja=false" }
```

Comment:
No, this is all no-op for Mono. Mono only supports one generator on Windows.

Reply:
Given that the switch is in the global script, maybe add "this is a no-op on Mono" to the help string.

Reply:
Why does mono only support one generator in windows?

Reply:
I don't know. I think they just use Ninja and don't use MSBuild because that's what matches their workflow.

We're one of a very few number of more modern OSS projects I know of that doesn't use only the Ninja generator (even xplat). Even the MSVC teams use only Ninja for their open source projects.

Reply:
Ah if Mono supports Ninja only that's absolutely fine. I thought it was the other way around.